src/main/java/org/eclipse/golo/compiler/ir/GoloFunction.java

```diff
@@ -334,4 +335,30 @@ public void walk(GoloIrVisitor visitor) {
   protected void replaceElement(GoloElement original, GoloElement newElement) {
     throw cantReplace();
   }
+
+  @Override
+  public int hashCode() {
+    int hash = 5;
+    hash = 89 * hash + Objects.hashCode(this.name);
+    hash = 89 * hash + Objects.hashCode(this.getArity());
+    return hash;
+  }
+
```

Comment:
IDE do it right usually, especially when you specify which fields are nullable :+1: